libcontainer/container.go

```diff
@@ -60,8 +60,8 @@ type State struct {
 //
 // Each container is thread-safe within the same process. Since a container can
 // be destroyed by a separate process, any function may return that the container
-// was not found.
-type Container interface {
+// was not found. CommonContainer includes methods that are platform agnostic.
+type CommonContainer interface {
```

Comment:
Can anyone think of other names besides `Common*`.  @jhowardmsft not that its bad, I'm just trying to think of other options for what we are trying to do here.